app/components/TablePaginationControls.tsx

```diff
@@ -47,8 +58,7 @@ export default function TablePaginationControls({
           }}
         >
           <AlignedLabel>
-            <FaCircleArrowLeft />
-            <span className="ms-2">Prev</span>
+            <FaAngleLeft />
           </AlignedLabel>
         </Button>
         <Button
```

Comment:
I'm not entirely sure why this label is wrapped in a button, but I can dig it if it's what makes this look like it does inside this `<AlignedLabel>`. It feels like it wants to be a div or a span maybe?

But anyway -- the real thing I want to propose here is likely beyond the scope of this issue, but may be worth spinning off into the backlog:

What would you think about a little "hidden functionality" where if you click on the `Page _n_` label, you get a drop-down from which you can select which page you want to jump to. The functionality is non-critical to the use of the page, so I feel that it's entirely fine if we don't indicate that it's possible, and just let our power-users discover it in their own time and way.